cuebot/src/main/java/com/imageworks/spcue/service/JobSpec.java

```diff
@@ -512,31 +512,25 @@ private void determineMinimumMemory(BuildableJob buildableJob,
         long minMemory;
         String memory = layerTag.getChildTextTrim("memory").toLowerCase();
 
-        try {
```

Comment:
Hi @DiegoTavares 

Why the external `try ... catch` was removed. Consider re-adding the missing external `try ... catch`.

```java
try {
    minMemory = convertMemoryInput(memory);

    // Some quick sanity checks to make sure memory hasn't gone
    // over or under reasonable defaults.
    if (minMemory> Dispatcher.MEM_RESERVED_MAX) {
        throw new SpecBuilderException("Memory requirements exceed " +
                "maximum. Are you specifying the correct units?");
    }
    else if (minMemory < Dispatcher.MEM_RESERVED_MIN) {
        logger.info(buildableJob.detail.name + "/" + layer.name +
                "Specified too little memory, defaulting to: " +
                Dispatcher.MEM_RESERVED_MIN);
        minMemory = Dispatcher.MEM_RESERVED_MIN;
    }

    buildableLayer.isMemoryOverride = true;
    layer.minimumMemory = minMemory;

} catch (Exception e) {
    logger.info("Setting memory for " +
            buildableJob.detail.name + "/" + layer.name +
            " failed, reason: " + e + ". Using default.");
    layer.minimumMemory = Dispatcher.MEM_RESERVED_DEFAULT;
}
```

Reply:
My change removes the `throw` statement that was being caught by the `catch` that got removed. Did you find any reason on this section for a broad `Exception` catch?